app/routes/users/components/UserConfirmation.tsx

```diff
@@ -89,91 +95,94 @@ const UserConfirmation = ({
+                  component={TextInput.Field}
+                />
+                <Field
+                  name="firstName"
+                  placeholder="Fornavn"
+                  label="Fornavn"
+                  autocomplete="given-name additional-name"
+                  component={TextInput.Field}
+                />
+                <Field
+                  name="lastName"
+                  label="Etternavn"
+                  placeholder="Etternavn"
+                  autocomplete="family-name"
+                  component={TextInput.Field}
+                />
+                <MultiSelectGroup label="Kjønn" name="gender">
+                  <Field
+                    name="genderMan"
+                    label="Mann"
+                    component={RadioButton.Field}
+                    inputValue="male"
+                  />
+                  <Field
+                    name="genderWoman"
+                    label="Kvinne"
+                    component={RadioButton.Field}
+                    inputValue="female"
+                  />
+                  <Field
+                    name="genderOther"
+                    label="Annet"
+                    component={RadioButton.Field}
+                    inputValue="other"
+                  />
+                </MultiSelectGroup>
+                <Field
+                  name="allergies"
+                  placeholder="Allergier"
+                  label="Matallergier/preferanser"
+                  component={TextInput.Field}
+                />
 
-          <Field
-            label="Telefonnummer"
-            name="phoneNumber"
-            autocomplete="tel"
-            component={PhoneNumberInput.Field}
-          />
-          <Button submit>Registrer bruker</Button>
-        </Form>
+                <Field
+                  label="Telefonnummer"
+                  name="phoneNumber"
+                  autocomplete="tel"
+                  component={PhoneNumberInput.Field}
+                />
+                {spySubmittable((submittable) => (
+                  <Button submit disabled={!submittable}>
+                    Registrer bruker
+                  </Button>
+                ))}
+              </form>
+            );
+          }}
+        </LegoFinalForm>
       </div>
     </Container>
   );
 };
 
-const validate = createValidator({
-  username: [required()],
-  password: [required()],
-  retypePassword: [required(), sameAs('password', 'Passordene er ikke like')],
-  gender: [required()],
-});
-const asyncValidate = createAsyncValidator({
-  password: [validPassword()],
-});
-
-const onSubmit = (data, dispatch, { token, createUser }) =>
-  createUser(token, data);
+const validate = createValidator(
+  {
+    username: [required()],
+    password: [required(), validPassword()],
+    retypePassword: [required(), sameAs('password', 'Passordene er ikke like')],
+    gender: [required()],
+  },
+  undefined,
+  true
```

Comment:
What is this?

Reply:
apparently some validators does not exist in a single field, so rawValidators are ran without a field argument (but is unused here)